entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
We're not consistent everywhere.  For beginner tutorials we often add a using as a step in the text.  Some topics aimed at relatively advanced users don't mention the usings, assuming the reader knows or can easily find them.  Some topics show you initially how to add them using VS refactoring, then presume you'll use it after that.

* In this article I don't think the note about using is necessary as this is a reference doc for someone who has already started working with EF.
* Could include the using by rendering two snippets, using snippet immediately followed by code snippet.
* Top of page would be better in a doc like this where you would typically go directly to the section you want rather than read sequentially the whole doc, possibly missing something at the end.
* You get a really obtrusive note format with [!NOTE], so Rick and I sometimes use plain text "Note:" if the nature of the content doesn't require such a loud shout; I usually bold the word **Note**.